From 524bc0710f6dbbbb6b8135253f03ce5e0059da02 Mon Sep 17 00:00:00 2001
From: rpm-build <rpm-build>
Date: Tue, 4 Sep 2018 09:58:57 -0400
Subject: [PATCH] data: drop app icon styling

classic session doesn't show an app icon in the app menu, so
putting a drop shadow around where it would be creates screen
artifacts.

This commit drops the styling of the app icon that doesn't exist.
---
 data/gnome-shell-sass/_common.scss | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/data/gnome-shell-sass/_common.scss b/data/gnome-shell-sass/_common.scss
index 1ceadf4..3cce6c1 100644
--- a/data/gnome-shell-sass/_common.scss
+++ b/data/gnome-shell-sass/_common.scss
@@ -742,140 +742,130 @@ StScrollBar {
 
   #panelLeft, #panelCenter { // spacing between activities<>app menu and such
     spacing: 4px;
   }
 
   .panel-corner {
     -panel-corner-radius: $panel-corner-radius;
     -panel-corner-background-color: rgba(0, 0, 0, 0.35);
     -panel-corner-border-width: 2px;
     -panel-corner-border-color: transparent;
 
     &:active, &:overview, &:focus {
       -panel-corner-border-color: lighten($selected_bg_color,5%);
     }
 
     &.lock-screen, &.login-screen, &.unlock-screen {
       -panel-corner-radius: 0;
       -panel-corner-background-color: transparent;
       -panel-corner-border-color: transparent;
     }
   }
 
   .panel-button {
     -natural-hpadding: 12px;
     -minimum-hpadding: 6px;
     font-weight: bold;
     color: #eee;
     text-shadow: 0px 1px 2px rgba(0, 0, 0, 0.9);
     transition-duration: 100ms;
 
-    .app-menu-icon {
-      -st-icon-style: symbolic;
-      margin-left: 4px;
-      margin-right: 4px;
-      //dimensions of the icon are hardcoded
-    }
-
     .panel-logo-icon {
       padding-right: .4em;
       icon-size: 1em;
     }
 
     .system-status-icon,
-    .app-menu-icon > StIcon,
     .popup-menu-arrow {
       icon-shadow: 0px 1px 2px rgba(0, 0, 0, 0.9);
     }
 
     &:hover {
       color: lighten($fg_color, 10%);
       text-shadow: 0px 1px 6px rgba(0, 0, 0, 1);
 
       .system-status-icon,
-      .app-menu-icon > StIcon,
       .popup-menu-arrow {
         icon-shadow: 0px 1px 6px rgba(0, 0, 0, 1);
       }
     }
 
     &:active, &:overview, &:focus, &:checked {
       // Trick due to St limitations. It needs a background to draw
       // a box-shadow
       background-color: rgba(0, 0, 0, 0.01);
       box-shadow: inset 0 -2px 0px lighten($selected_bg_color,5%);
       color: lighten($fg_color,10%);
 
       & > .system-status-icon { icon-shadow: black 0 2px 2px; }
     }
 
     .system-status-icon { icon-size: 1.09em; padding: 0 5px; }
     .unlock-screen &,
     .login-screen &,
     .lock-screen & {
       color: lighten($fg_color, 10%);
       &:focus, &:hover, &:active { color: lighten($fg_color, 10%); }
     }
   }
 
   .panel-status-indicators-box,
   .panel-status-menu-box {
     spacing: 2px;
   }
 
   // spacing between power icon and (optional) percentage label
   .power-status.panel-status-indicators-box {
     spacing: 0;
   }
 
   .screencast-indicator { color: $warning_color; }
 
   &.solid {
     background-color: black;
     /* transition from transparent to solid */
     transition-duration: 300ms;
 
     .panel-corner {
       -panel-corner-background-color: black;
     }
 
     .panel-button {
       color: #ccc;
       text-shadow: none;
 
       &:hover, &:active, &:overview, &:focus, &:checked {
         color: lighten($fg_color, 10%);
       }
     }
 
     .system-status-icon,
-    .app-menu-icon > StIcon,
     .popup-menu-arrow {
       icon-shadow: none;
     }
   }
 }
 
   // calendar popover
   #calendarArea {
     padding: 0.75em 1.0em;
   }
 
   .calendar {
     margin-bottom: 1em;
   }
 
     .calendar,
     .datemenu-today-button,
     .datemenu-displays-box,
     .message-list-sections {
       margin: 0 1.5em;
     }
 
     .datemenu-calendar-column { spacing: 0.5em; }
     .datemenu-displays-section { padding-bottom: 3em; }
     .datemenu-displays-box { spacing: 1em; }
 
     .datemenu-calendar-column {
       border: 0 solid lighten($bg_color,5%);
       &:ltr { border-left-width: 1px; }
       &:rtl { border-right-width: 1px; }
-- 
2.17.1

